Assets/Plugins/Source/Editor/Windows/EOSSettingsWindow.cs

```diff
@@ -399,13 +400,15 @@ protected override void RenderWindow()
             GUIEditorUtility.AssigningBoolField("Save JSON in 'Pretty' Format", ref prettyPrint, 190);
             if (GUILayout.Button("Save All Changes"))
             {
-                SaveToJSONConfig(prettyPrint);
+                // Save NOTE: if this times out, then there will be problems.
+                Save(prettyPrint).Wait();
```

Comment:
Again, make sure that exceptions are handled appropriately, and that there are reasonable.